CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+## [0.7.0] - 2020-07-22
+
+This release has minor backwards incompatible for anyone that directly used
+datafiles. The top level and filesystem APIs (assemble, iterate, partitioned,
+etc) are unchanged.
+
+### Added
+
+- LocalDataFile that implements datafile for local paths
+- GCSNativeDataFile that implements datafile for GCS paths using GCS python blob API
```

Comment:
Does this support parallel downloading with their blob API?

Reply:
This does not yet support it, we'd have to implement our own threading (or multiprocessing) here. This PR will make it much easier to add that support in the future